plugins/opa/examples/hpa-required/workloads.yaml

```diff
@@ -0,0 +1,7 @@
+targets:
+- apiGroups: ["apps"]
+  kinds: ["DaemonSet", "Deployment", "StatefulSet"]
```

Comment:
Let's bring this down to just deployments. An hpa on the others doesn't feel like it makes sense

Reply:
Hah very good point